test/governance/Governor.test.js

```diff
@@ -83,577 +100,517 @@ contract('Governor', function (accounts) {
       shouldBehaveLikeERC6372(mode);
 
       it('deployment check', async function () {
-        expect(await this.mock.name()).to.be.equal(name);
-        expect(await this.mock.token()).to.be.equal(this.token.address);
-        expect(await this.mock.votingDelay()).to.be.bignumber.equal(votingDelay);
-        expect(await this.mock.votingPeriod()).to.be.bignumber.equal(votingPeriod);
-        expect(await this.mock.quorum(0)).to.be.bignumber.equal('0');
-        expect(await this.mock.COUNTING_MODE()).to.be.equal('support=bravo&quorum=for,abstain');
+        expect(await this.mock.name()).to.equal(name);
+        expect(await this.mock.token()).to.equal(this.token.target);
+        expect(await this.mock.votingDelay()).to.equal(votingDelay);
+        expect(await this.mock.votingPeriod()).to.equal(votingPeriod);
+        expect(await this.mock.quorum(0)).to.equal(0n);
+        expect(await this.mock.COUNTING_MODE()).to.equal('support=bravo&quorum=for,abstain');
       });
 
       it('nominal workflow', async function () {
         // Before
-        expect(await this.mock.proposalProposer(this.proposal.id)).to.be.equal(constants.ZERO_ADDRESS);
-        expect(await this.mock.hasVoted(this.proposal.id, owner)).to.be.equal(false);
-        expect(await this.mock.hasVoted(this.proposal.id, voter1)).to.be.equal(false);
-        expect(await this.mock.hasVoted(this.proposal.id, voter2)).to.be.equal(false);
-        expect(await web3.eth.getBalance(this.mock.address)).to.be.bignumber.equal(value);
-        expect(await web3.eth.getBalance(this.receiver.address)).to.be.bignumber.equal('0');
+        expect(await this.mock.proposalProposer(this.proposal.id)).to.equal(ethers.ZeroAddress);
+        expect(await this.mock.hasVoted(this.proposal.id, this.owner)).to.be.false;
+        expect(await this.mock.hasVoted(this.proposal.id, this.voter1)).to.be.false;
+        expect(await this.mock.hasVoted(this.proposal.id, this.voter2)).to.be.false;
+        expect(await ethers.provider.getBalance(this.mock)).to.equal(value);
+        expect(await ethers.provider.getBalance(this.receiver)).to.equal(0n);
 
-        expect(await this.mock.proposalEta(this.proposal.id)).to.be.bignumber.equal('0');
-        expect(await this.mock.proposalNeedsQueuing(this.proposal.id)).to.be.equal(false);
+        expect(await this.mock.proposalEta(this.proposal.id)).to.equal(0n);
+        expect(await this.mock.proposalNeedsQueuing(this.proposal.id)).to.be.false;
 
         // Run proposal
-        const txPropose = await this.helper.propose({ from: proposer });
-
-        expectEvent(txPropose, 'ProposalCreated', {
-          proposalId: this.proposal.id,
-          proposer,
-          targets: this.proposal.targets,
-          // values: this.proposal.values,
```

Comment:
Any reason why this was commented out before and now it's included?

Reply:
We used to not be able the check array of BN. With the migration to bigint we can check it.

Also, ethers+chai requires all events members to be included. we would put `anyValue`, but since we have the value its better to check it.